README.md

```diff
@@ -26,7 +26,7 @@ All image processing functions provided operate on types that implement the `Gen
 | BMP    | Yes | Rgb8, Rgba8, Gray8, GrayA8 |
 | ICO    | Yes | Yes |
 | TIFF   | Baseline(no fax support) + LZW + PackBits | Rgb8, Rgba8, Gray8 |
-| WebP   | Yes | No |
+| WebP   | Yes | Rgb8, Rgba8 |
```

Comment:
Might be worth adding asterisks to denote the cases that require a C dependency that isn't enabled by default

Reply:
Good point, is this okay?

https://github.com/cycraig/image/blob/master/README.md

| Format | Decoding | Encoding |
| ------ | -------- | -------- |
| WebP   | Yes | Rgb8, Rgba8 \* |
| AVIF   | Only 8-bit \** | Lossy |

- \* requires the `webp-encoder` feature, uses the libwebp C library.
- \** requires the `avif-decoder` feature, uses the libdav1d C library.